var/spack/repos/builtin/packages/py-voluptuous/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyVoluptuous(PythonPackage):
+    """Voluptous, despite the name, is a Python data validation library."""
+    homepage = "https://github.com/alecthomas/voluptuous"
+    url      = "https://github.com/alecthomas/voluptuous/archive/0.11.5.tar.gz"
+
+    version('0.11.5', sha256='01adf0b6c6f61bd11af6e10ca52b7d4057dd0be0343eb9283c878cf3af56aee4')
+
+    depends_on('python@2.7:,3.6:', type=('build', 'run'))
```

Comment:
I would actually get rid of this line. There is nothing like `python_requires` that explicitly says the package won't work for older versions of Python.

Also, you're missing a setuptools build dependency.